modules/web-console-configuration.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * web_console/configuring-web-console.adoc
+
+[id="web-console-configuration_{context}"]
+= Configuring the web console
+
+You can configure image registry settings by editing the
+`image.config.openshift.io/cluster` resource.
+
+* Edit the `consoles.operator.openshift.io resource:
++
+----
+$ oc edit consoles.operator.openshift.io
+----
++
+The following example displays the parameters from this resource that you can
+modify:
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Console
+metadata:
+  name: cluster
+spec:
+  managementState: Managed <1>
+  authentication:
+    logoutRedirect: "" <2>
+status:
+  consoleURL: "" <3>
+----
+<1> Specify the state of the console, `Managed` or `Disabled`. If the value is
+`Managed`, then the web console is enabled. Set it to `Disabled` to disable
+the console.
+<2> Specify the URL of the page to load when a user logs out of the web console.
+If you do not specify a value, the user returns to the login page for the
+web console. Specifying a `logoutRedirect` URL allows your users to perform
+single log out (SLO) through the identity provider to destroy their single
+sign-on session.
+<3> The web console URL. Do not modify this value.
```

Comment:
I don't know that you have to say `Do not modify this value` as it will get fixed if a user tampers.  Expressing that this is the canonical URL of the web console for the cluster is probably sufficient.